pkg/ccl/backupccl/show.go

```diff
@@ -636,14 +645,15 @@ func checkBackupFiles(
 }
 
 type backupInfo struct {
-	collectionURI string
-	defaultURIs   []string
-	manifests     []backuppb.BackupManifest
-	subdir        string
-	localityInfo  []jobspb.RestoreDetails_BackupLocalityInfo
-	enc           *jobspb.BackupEncryptionOptions
-	kmsEnv        cloud.KMSEnv
-	fileSizes     [][]int64
+	collectionURI      string
+	defaultURIs        []string
+	manifests          []backuppb.BackupManifest
+	layerToIterFactory backupinfo.LayerToBackupManifestFileIterFactory
```

Comment:
nit: a quick comment about what this field is?

Reply:
done, added comment.